### PR TITLE
[refactor](wal) move group commit load content length to runtime state

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: be/src/runtime/group_commit_mgr.h [hunk 1]
```diff
@@ -153,9 +154,6 @@ class GroupCommitMgr {
                                       const UniqueId& load_id,
                                       std::shared_ptr<LoadBlockQueue>& load_block_queue,
                                       int be_exe_version);
-    Status update_load_info(TUniqueId load_id, size_t content_length);
```
Comment (mymeiyi) [inline be/src/runtime/group_commit_mgr.h:156]: should also remove `_load_id_to_content_length_map` and the lock

File: be/src/runtime/group_commit_mgr.cpp [hunk 2]
```diff
@@ -487,31 +487,24 @@ Status LoadBlockQueue::close_wal() {
 
 bool LoadBlockQueue::has_enough_wal_disk_space(
         const std::vector<std::shared_ptr<vectorized::Block>>& blocks, const TUniqueId& load_id,
-        bool is_blocks_contain_all_load_data) {
+        bool is_blocks_contain_all_load_data, size_t content_length) {
     size_t blocks_size = 0;
```
Comment (mymeiyi) [inline be/src/runtime/group_commit_mgr.cpp:491]: It's better to change the function to `bool LoadBlockQueue::has_enough_wal_disk_space(size_t pre_allocated)`

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by mymeiyi** [comment]
run p0

**Comment by Yukang-Lian** [comment]
run buildall

**Review by mymeiyi — APPROVED** [review]
LGTM

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**